drivers/lora/Kconfig.rylrxxx

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2024 David Ullmann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config LORA_RYLRXXX
+	bool "Reyax LYLR driver"
+	default n
+	depends on DT_HAS_REYAX_RYLRXXX_ENABLED
+	select MODEM
+	select MODEM_CONTEXT
+	select MODEM_CMD_HANDLER
+	select MODEM_IFACE_UART
+	select UART_USE_RUNTIME_CONFIGURE
+	help
+		Enable the Reyax LYLR driver
```

Comment:
help text should be indented by 2 spaces from the rest of the text, take a look at the other upstream Kconfig files for examples.